autofill/autofill-impl/build.gradle

```diff
@@ -33,6 +33,9 @@ dependencies {
     implementation project(path: ':device-auth-api')
     implementation project(path: ':browser-api')
     implementation project(path: ':autofill-store')
+    implementation project(path: ':statistics')
+    implementation project(path: ':app')
+    implementation project(path: ':app')
```

Comment:
The IDE added these during my attempts at importing the `AppPixelName` stuff into my file. Not sure what's going on and if they are needed.

Reply:
Keep `implementation project(path: ':statistics')` but kill the other two; they shouldn't be added